rubocop-rspec-rails.gemspec

```diff
@@ -32,12 +29,11 @@ Gem::Specification.new do |spec|
   spec.extra_rdoc_files = ['MIT-LICENSE.md', 'README.md']
 
   spec.metadata = {
-    'changelog_uri' => 'https://github.com/rubocop/rubocop-rspec/blob/master/CHANGELOG.md',
-    'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec/',
+    'changelog_uri' => 'https://github.com/rubocop/rubocop-rspec_rails/blob/master/CHANGELOG.md',
+    'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec_rails/',
     'rubygems_mfa_required' => 'true'
   }
 
   spec.add_runtime_dependency 'rubocop', '~> 1.40'
-  spec.add_runtime_dependency 'rubocop-capybara', '~> 2.17'
-  spec.add_runtime_dependency 'rubocop-factory_bot', '~> 2.22'
+  spec.add_runtime_dependency 'rubocop-rspec', '~> 2.27.1'
```

Comment:
I am not sure we want this line.

Reply:
For example, https://github.com/rubocop/rubocop-rspec/blob/master/lib/rubocop/rspec/language.rb thinks that RuboCop RSpecRails should be usable. What do you think?

Reply:
Removed in https://github.com/ydah/rubocop-rspec_rails/pull/1/commits/e0cd330f08d3f9a015b770a07ccb9ba92248b4ad